marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/endpoints/collector.sjs

```diff
@@ -69,19 +69,18 @@ if (!query) {
   fn.error(null, "RESTAPI-SRVEXERR", Sequence.from([404, "Not Found", "The collector query was empty"]));
 }
 
-query = new CollectorLib().prepareSourceQuery(combinedOptions, stepDefinition);
-
+const javascript = new CollectorLib().prepareSourceQuery(combinedOptions, stepDefinition);
```

Comment:
can we have a better variable name for "javascript" here?

Reply:
It's the same name as the argument for xdmp.eval - http://docs.marklogic.com/xdmp.eval .